src/main/resources/application.properties

```diff
@@ -1 +1,22 @@
 spring.application.name=quora.clone
+
+logging.level.org.springframework.web=debug
+
+
+spring.datasource.name=Quora
+spring.datasource.password=9926430125
+spring.datasource.url=jdbc:mysql://localhost:3306/Quora
+spring.datasource.username=root
+
+
+logging.level.org.springframework.r2dbc=DEBUG
+#spring.r2dbc.password=9926430125
+#spring.r2dbc.url=r2dbc:mysql://localhost:3306/Quora
+#spring.r2dbc.username=root
+
+# we can see sql statement being executed
+spring.jpa.show-sql=true
+
+
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
```

Comment:
why this is required?

Reply:
not required. I have commented out. `spring.jpa.generate-ddl=true` is added to auto generate table,schema by jpa.